config/routes.rb

```diff
@@ -19,7 +19,9 @@
 
     patch "/status", to: "boards#update_board_status", as: "status"
     patch "/close", to: "boards#update_board_closed", as: "board_closed"
-    resource :lists, only: :create
+    resources :lists, only: %i(create update)
     resources :add_members, only: %i(new edit)
+    post "list/changeposition", to: "lists#change_position", as: "position_list"
+    post "list/closed", to: "lists#closed_list", as: "closed_list"
```

Comment:
post is used to create new record, if should use method patch for update

Reply:
vâng để em sửa